buildkit/pgtap.yaml

```diff
@@ -7,6 +7,8 @@ source: https://github.com/theory/pgtap/archive/refs/tags/v1.3.1.tar.gz
 description: PostgreSQL Unit Testing Suite.
 keywords:
   - unit tests
+  - tdd
```

Comment:
`tdd` is a practice of writing tests first and this extension doesn't force that.